gdx/src/com/badlogic/gdx/math/Vector2.java

```diff
@@ -316,19 +316,40 @@ public float crs (float x, float y) {
 		return this.x * y - this.y * x;
 	}
 
+

```

Comment:
you can remove empty line (or use formater on modified blocs)

Reply:
thanks, missed that one. Unfortunately using formatter is still not an option, last time I ran it on modified code I had to change it all over again due to format differences between IDEs/formatters.